checkov/terraform/context_parsers/parsers/resource_context_parser.py

```diff
@@ -13,5 +15,19 @@ def get_entity_context_path(self, entity_block: Dict[str, Dict[str, Any]]) -> Li
         entity_name = next(iter(entity_block[entity_type]))
         return [entity_type, entity_name]
 
+    def enrich_definition_block(self, definition_blocks: List[Dict[str, Any]]) -> Dict[str, Any]:
+        for entity_block in definition_blocks:
+            entity_type, entity_value = next(iter(entity_block.items()))
+            entity_name, entity_config = next(iter(entity_value.items()))
+
+            self.context[entity_type][entity_name] = {
+                "start_line": entity_config[START_LINE],
+                "end_line": entity_config[END_LINE],
+                "code_lines": self.file_lines[entity_config[START_LINE] - 1: entity_config[END_LINE]],
+            }
```

Comment:
@gruebel Is this duplicated from [data_context_parser.py](https://github.com/bridgecrewio/checkov/pull/2760/files#diff-c9156a98f78acf8cd94baf468cc169483044003c4f8ccc81a673d84836db3c96)?